ibis/expr/types/maps.py

```diff
@@ -4,15 +4,16 @@
 
 from public import public
 
+import ibis
 import ibis.expr.operations as ops
+import ibis.expr.types as ir
 from ibis.common.deferred import deferrable
 from ibis.expr.types.generic import Column, Scalar, Value
 
 if TYPE_CHECKING:
     from collections.abc import Iterable, Mapping
 
-    import ibis.expr.types as ir
-    from ibis.expr.types.arrays import ArrayColumn
+    from ibis.expr import datatypes as dt
```

Comment:
Please follow the same import style as the other Ibis modules:

```suggestion
    import ibis.expr.datatype as dt
```

Reply:
oops, sorry this was blind acceptance of either vscode intellisense or copilor suggestsions.